src/erlang_ls_parser.erl

```diff
@@ -4,17 +4,19 @@
         , parse_file/1
         ]).
 
-%% TODO: Generate random filename
-%% TODO: Ideally avoid writing to file at all (require epp changes)
--define(TMP_PATH, <<"/tmp/erlang_ls_tmp">>).
-
 -spec parse(binary()) ->
    {ok, erlang_ls_tree:tree(), erlang_ls_tree:extra()} | {error, any()}.
 parse(Text) ->
   %% epp_dodger only works with source files,
   %% so let's use a temporary file.
-  ok = file:write_file(?TMP_PATH, Text),
-  parse_file(?TMP_PATH).
+  %% TODO: Ideally avoid writing to file at all (require epp changes)
+  TmpPath = tmp_path(),
```

Comment:
This was necessary to avoid the indexing intefering with the same temp file that another file was using.